Makefile

```diff
@@ -34,11 +33,8 @@ in/CodeList.csv: | in
 	wget -c https://raw.githubusercontent.com/datasets/un-locode/${UNLOCODEREVISION}/data/code-list.csv -O $@
 	awk 'NR>1' $@ > temp && mv temp $@
 
-$(LOCODECLI):
-	go build -o $(LOCODECLI) ./internal
-
-generate: in/airports.dat in/countries.dat in/continents.geojson in/SubdivisionCodes.csv in/CodeList.csv $(LOCODECLI) | $(LOCODEDB)
-	./$(LOCODECLI) generate \
+generate: in/airports.dat in/countries.dat in/continents.geojson in/SubdivisionCodes.csv in/CodeList.csv | $(LOCODEDB)
+	go run ./internal \
```

Comment:
i see
```
make: go: Permission denied
make: *** [Makefile:37: generate] Error 127
```
on my os which can be fixed via semicolon termination. Afaik this caused by `go` dir in `$PATH`. Does this work 4 u w/o `;`?

```
$ make --version
GNU Make 4.3
Built for x86_64-pc-linux-gnu
```

Reply:
I get the same output from `make --version` and I don't have this problem. Have you encountered any issues with previous versions of the file? It works both with and without `;`. Should I leave the `;`?

Reply:
>  Have you encountered any issues with previous versions of the file?

i didnt try tbh, will see

> Should I leave the `;`?

i'd say yeah: it breaks nothing while can heal cases w/ specific OS path like mine. This may be fixed in make 4.4